core/templates/components/button-directives/social-buttons.component.css

```diff
@@ -53,6 +53,15 @@
   margin-right: 6px;
   padding: 5px 6px;
 }
+.oppia-footer-social .oppia-footer-social-icons .oppia-github-follow {
+  background: #fff;
+  margin-right: 6px;
+  padding: 5px 6px;
+
```

Comment:
Remove extra newline!

Reply:
@shri3016 Can you please resolve this comment before we merge the PR?

Reply:
Hi @DubeySandeep,PTAL.
Thank you!